packages/cap-config-writer/src/cap-writer/package-json.ts

```diff
@@ -0,0 +1,156 @@
+import type { Package } from '@sap-ux/project-access';
+import { getCapCustomPaths } from '@sap-ux/project-access';
+import type { Editor } from 'mem-fs-editor';
+import path, { join } from 'path';
+import type { CdsUi5PluginInfo, CapServiceCdsInfo } from '../cap-config/types';
+import { enableCdsUi5Plugin, checkCdsUi5PluginEnabled, satisfiesMinCdsVersion, minCdsVersion } from '../cap-config';
+import type { Logger } from '@sap-ux/logger';
+import { t } from '../i18n';
+
+/**
+ * Retrieves the CDS watch script for the CAP app.
+ *
+ * @param {string} projectName - The project's name, which is the module name.
+ * @param {string} appId - The application's ID, including its namespace and the module name.
```

Comment:
Should we update to include, "if provided will be used to open the application instead of the project name, provide this option for example when using npm workspaces"